Mockingjay/NSURLSessionConfiguration.swift

```
@@ -9,28 +9,32 @@
 import Foundation
 
 let swizzleDefaultSessionConfiguration: Void = {
-  let defaultSessionConfiguration = class_getClassMethod(URLSessionConfiguration.self, #selector(getter: URLSessionConfiguration.default))
-  let mockingjayDefaultSessionConfiguration = class_getClassMethod(URLSessionConfiguration.self, #selector(URLSessionConfiguration.mockingjayDefaultSessionConfiguration))
+  guard
+    let defaultSessionConfiguration = class_getClassMethod(URLSessionConfiguration.self, #selector(getter: URLSessionConfiguration.default)),
+    let mockingjayDefaultSessionConfiguration = class_getClassMethod(URLSessionConfiguration.self, #selector(URLSessionConfiguration.mockingjayDefaultSessionConfiguration))
+    else { return }
```

I don't think this should silently return otherwise when setting up the swizzling users won't know if it failed.

Probably better to fatalError stating the problem.